scraper/spiders/meeting_spider.py

```diff
@@ -86,8 +86,8 @@ def parse(self, response):
             marker_hearing = meeting.get('hearing')
             marker_time = marker_time.replace(".",":")
             marker_url = response.url
-            marker_open_again = meeting.get('oa')
-            if marker_open_again == 'True':
+            marker_meeting_status = meeting.get('ms')
+            if marker_meeting_status in ['5']:
```

Comment:
```
# See GitHub issue #225
MEETING_STATUS_CODES = {"Back_Open_Again"    : "5",
                        "Online"             : "4",
                        "Temporarily Closed" : "3"}

if marker_meeting_status == MEETING_STATUS_CODES["Back_Open_Again"]: 
```

Reply:
Could we use a lookup here, instead of "5"? I thought it might make it clearer.

Reply:
Yes good call much better